x/staking/client/cli/query.go

```diff
@@ -113,22 +113,36 @@ $ %s query staking validators
 				return err
 			}
 
-			resKVs, _, err := clientCtx.QuerySubspace(types.ValidatorsKey, types.StoreKey)
+			queryClient := types.NewQueryClient(clientCtx)
+
+			// Query all validators by quering validators for all statuses.
+			bonded, err := queryClient.Validators(context.Background(), &types.QueryValidatorsRequest{
+				Status: types.BondStatusBonded,
```

Comment:
I believe that if the status is empty, it will return all the validators, which would save up some code below.

Reply:
Good point.